resources/views/events/index.blade.php

```diff
@@ -20,27 +20,6 @@
           </div>
       @endif
 
-      <div class="row mb-30">
-          <div class="col-12 col-md-12">
-              <div class="d-flex align-items-center">
-                  <h1 class="mb-0 mr-30">
-                      @lang('events.events')
```

Comment:
This seems to have lost the title, doodle, create event button, from the Events page  (/party) - presumably unintentional?

Reply:
Yes, you're entirely right.